### PR TITLE
Support custom CA for sidecars, ingressgateway and Prometheus

### PR DESCRIPTION
**Pull request by myidpt**
[ ] Configuration Infrastructure
[ ] Docs
[ X ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: operator/cmd/mesh/testdata/manifest-generate/output/all_on.golden-show-in-gh-pull-request.yaml [hunk 1]
```diff
@@ -9178,6 +9180,10 @@ data:
         - name: {{ $key }}
           value: "{{ $value }}"
         {{- end }}
+        {{- range $key, $value := .Values.global.proxyEnvVars }}
```
Comment (howardjohn) [inline operator/cmd/mesh/testdata/manifest-generate/output/all_on.golden-show-in-gh-pull-request.yaml]: we already have this, see immediate line above

**Review by howardjohn — CHANGES_REQUESTED** [review]

File: operator/cmd/mesh/testdata/manifest-generate/output/all_on.golden-show-in-gh-pull-request.yaml (hunk 1 above)
Comment (myidpt) [reply operator/cmd/mesh/testdata/manifest-generate/output/all_on.golden-show-in-gh-pull-request.yaml]: Sure. But I had a hard time to set it on istioctl:
out/linux_amd64/istioctl manifest apply --set values.global.tag=$TAG \
--set proxyConfig.proxyMetadata.foo=bar
doesn't work. Do you know how to set it?

Comment (myidpt) [reply operator/cmd/mesh/testdata/manifest-generate/output/all_on.golden-show-in-gh-pull-request.yaml]: @ostromart

Comment (myidpt) [reply operator/cmd/mesh/testdata/manifest-generate/output/all_on.golden-show-in-gh-pull-request.yaml]: The section above works. I'll remove this part.

**Review by howardjohn — APPROVED** [review]

**Pull request merged**